### PR TITLE
[Frontend_v2] Challenge detail UI fix

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
**Changes introduced through this PR**

- [x] Improved the UI for challenge detail.

- [x] Toggle button of `sort by latest` changed to checkbox.

**Before UI Improvement:**
beforeUIfix

**After UI Improvement**
UIfix

cc: @Sanji515 @RishabhJain2018 @deshraj @Shekharrajak @lunayach

### CODE REVIEW

File: frontend_v2/src/app/components/challenge/challenge.component.html [hunk 1]
```diff
@@ -15,76 +15,66 @@
...
                                                     (click)="deleteChallenge()">
-                                                    <i class="fa fa-trash red-text"
+                                                    <i class="fa fa-trash red-text fs-16"
                                                         aria-hidden="true"></i>
                                                 </a>
                                             </span>
-                                            <br>
-                                            <span style="line-height: 1.5"
+                                            </h4>
+                                            <span style="line-height: 1.5" class="fw-light fs-16"
```
Comment (RishabhJain2018) [inline frontend_v2/src/app/components/challenge/challenge.component.html:34]: Hey @Kajol-Kumari, Let's not use inline CSS. Can you please create a class and add this.

File: frontend_v2/src/app/components/challenge/challenge.component.html [hunk 2]
```diff
@@ -15,76 +15,66 @@
...
+                                                    class="text-light-black">
+                                                    Organized by:</strong><a
                                                     class="orange-text"
                                                     href="{{challenge['creator']['team_url']}}"
                                                     target="_blank">
                                                     {{challenge['creator']['team_name']}}
                                                 </a>
                                             </span>
-                                            <span style="line-height: 1.5"
+                                            <span style="line-height: 1.5" class="fw-light fs-16"
```
Comment (RishabhJain2018) [inline frontend_v2/src/app/components/challenge/challenge.component.html:44]: Same here.

File: frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.html [hunk 3]
```diff
@@ -3,21 +3,21 @@
 		<div class="ev-pad">
 			<div class="ev-md-container ev-card-panel">
 				<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10 content">
-					<span class="fw-semibold fs-18">
-						<strong class="fw-semibold">Evaluation Criteria</strong>
-					</span>
-					<div class="evaluation-details" id="challenge-evaluation"></div>
+					<!-- <span class="fw-light"> -->
+						<h5 class="fw-light">Evaluation Criteria</h5> <br>
+					<!-- </span> -->
```
Comment (Sanji515) [inline frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.html:8]: Please remove these comments too.

File: frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.html [hunk 4]
```diff
@@ -27,15 +27,15 @@
     <div class="tnc-pad">
 			<div class="ev-md-container ev-card-panel">
 				<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10 content">
-					<span class="fw-semibold fs-18">
-						<strong class="fw-semibold">Terms And Conditions</strong>
-					</span>
-					<div class="terms-and-conditions" id="challenge-tnc"></div>
+					<!-- <span class="fw-light fs-18"> -->
+						<h5 class="fw-light">Terms And Conditions</h5> <br>
+					<!-- </span> -->
```
Comment (Sanji515) [inline frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.html:32]: Same here.

**Pull request merged**